jax/experimental/jax2tf/jax2tf.py

```diff
@@ -834,7 +834,7 @@ def _convert_value(val, aval):
   kept_args_avals = [aval for i, aval in enumerate(exported.in_avals) if i in exported.module_kept_var_idx]
   kept_args_flat_tf = [atf for i, atf in enumerate(args_flat_tf) if i in exported.module_kept_var_idx]
 
-  version = exported.xla_call_module_version
+  version = exported.serialization_version
```

Comment:
Since we have multiple versions now (xla_call_module_version, exported.serializtion_version), which doc we have the exact definition and about their relationship ?

Reply:
I renamed xla_call_module_version to serialization_version here. The only documentation is the [docstring of the Exported data structure](https://github.com/google/jax/blob/83d99bbb17fb0a03e14b190485c7e264481790c1/jax/experimental/jax2tf/jax_export.py#L117).

If you are asking about the relationship between tf.XlaCallModule versioning and jax_export versioning, there is some documentation [here](https://github.com/google/jax/blob/83d99bbb17fb0a03e14b190485c7e264481790c1/jax/experimental/jax2tf/jax_export.py#L117).

Reply:
Thanks a lot.  The 2nd link is the same as the 1st one. I guess you mean this one https://github.com/google/jax/blob/main/jax/experimental/jax2tf/README.md#native-serialization-versions ?

Thanks a lot for your explain. LGTM.